server/flyway/sql/V72__grant_api_user_read_access_application_client.sql

```diff
@@ -0,0 +1,4 @@
+-- Grant read access on fam_application_client to api user
+GRANT SELECT ON app_fam.fam_application_client TO ${api_db_username};
```

Comment:
This is needed for api db user to lookup application from Cognito client id for api call authorization check.